### PR TITLE
Update the `dev` branch for 2025.01 release

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-234.surge.sh

File: publish.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ site:
 content:
   sources:
   - url: ./
-    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','HEAD']
+    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','5.x','HEAD']
```
Comment (renetapopova) [inline publish.yml:9]:
```suggestion
    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','5.26','HEAD']
```

Comment (NataliaIvakina) [reply publish.yml:9]: Are we going to rename the branch?

Comment (NataliaIvakina) [reply publish.yml:9]: I think these branches are the Github branches, not versions of the manual.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-234.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-234.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-234.surge.sh

**Pull request merged**